packages/api-v2/types.ts

```diff
@@ -2163,6 +2163,26 @@ type TPrivateSocketEndpoints = {
         req_id?: number;
         [k: string]: unknown;
     };
+    trading_platform_status: {
+        request: {
+            /**
+             * Must be `1`
+             */
+            trading_platform_status: 1;
+        };
+        response: {
+            trading_platform_status: {
+                /**
+                 * cfd platforms
+                 */
+                platform: 'ctrader' | 'dxtrade' | 'mt5';
+                /**
+                 * possible platform statuses.
+                 */
+                status: 'active' | 'unavailable' | 'maintenance';
+            };
+        };
+    };
```

Comment:
Dont think you need to add this to PrivateEndpoint since you already putting the types in SocketEndpoints. TSocketEndpoints already has TPrivateSocketEndpoints in it.